lighthouse-core/gather/gatherers/trace-elements.js

```diff
@@ -36,6 +36,7 @@ function setAttributeMarker(metricName) {
       nodeLabel: getNodeLabel(elem), // eslint-disable-line no-undef
       // @ts-ignore - put into scope via stringification
       snippet: getOuterHTMLSnippet(elem), // eslint-disable-line no-undef
+      boundingRect: JSON.parse(JSON.stringify(elem.getBoundingClientRect())),
```

Comment:
should we be doing this on behalf of `evaluateAsync` calls?

Reply:
no, just objects that have getters